kbi/0012/index.rst

```diff
@@ -0,0 +1,56 @@
+.. index::
+   single: git-annex; encryption
+
+KBI0012: Annex encryption & gpg pinentry
```

Comment:
Maybe we can make it more clear in the title that this KBI is about resolving an issue, and not, e.g., about how to set encryption up? Maybe its also useful to cross-reference your usecase in the handbook as a pointer: http://handbook.datalad.org/en/latest/usecases/encrypted_annex.html

Reply:
I agree

Reply:
Good point - updated.